app/scripts/metamask-controller.test.js

```diff
@@ -1348,14 +1348,20 @@ describe('MetaMaskController', () => {
         await metamaskController._onKeyringControllerUpdate({
           keyrings: [
             {
-              accounts: ['0x1', '0x2'],
+              accounts: [
```

Comment:
To avoid duplicating this, we could add a `const accounts = [...];` and use int both the  state update and the `expect` WDYT?